### ISSUE TITLE
Create “shop_list_page”

### ISSUE THREAD

**Issue by huhn511** [nileorg/nile.shopping#6]
Note:  Selecting a category or the filter by name, a preview of the most successful product (respecting the filter) of each store will be displayed under the store box

![shops_list_page](https://user-images.githubusercontent.com/10562055/56113406-0e1f2f00-5f5e-11e9-981e-089b6be444b5.jpg)

**Comment by scorpion9979** [comment]
I've already done a bit of work on this. I did a lot of experimenting, and I think Vue with Buefy make a killer combination.

**Comment by scorpion9979** [comment]
![image](https://user-images.githubusercontent.com/4002635/56205747-4c4a4a80-6053-11e9-850e-c8fd45d09c48.png)

**Comment by huhn511** [comment]
Nice beginning work! 

What do you think about this state like Online/Offline. It's possible to buy also in Offline stores, right? If yes, we should remove it, to not confuse the users.

**Comment by scorpion9979** [comment]
It is actually possible, but I think the difference is that the orders won't be processed immediately. We can remove it, no problem.

**Comment by marcocastignoli** [comment]
Yes, I think the field would just confuse the user, but removing it is so simple. So for now we can just leave it there :)

What do you think about the feature of displaying a preview of the product when filtering by name?

**Comment by rep0x** [comment]
Okay this is pretty cool, for the first step (MVP) we wanted to exclude that part of having cities / ecosystems because the process could be closed within one ecosystem. But this is actually pretty important as soon as we move forward. In my head the cities/ecosystems are kind of like discord servers. So clicking the button on the top left (menu-btn) opens more less the same navigation than discord has on the left side.- 

[TODO] Wireframe for ecosystem / city navigation

For the home page (of a representative ecosystem) we thought of something like different horizontal sliders among themselves each with different topics. For example "Seasonal products" - "Popular shops" - "Recently bought" - "Hot deals" and so on.

The second tab would be a discover page where you can discover products or shops and filter them. So that would be what you have designed there in the issue. The question here would be what exactly the filter would do. I assume it is not supposed to be a search function because that would be something I would expand to the whole ecosystem.
